ai21/clients/studio/resources/studio_constants.py

```diff
@@ -0,0 +1,6 @@
+CHAT_DEFAULT_NUM_RESULTS = 1
```

Comment:
Since its under the studio package, no need to call the file `studio_constants.py` - `constants.py` should suffice :)